test/objects.test.ts

```diff
@@ -454,4 +454,56 @@ describe("Objects", () => {
       >
     >;
   });
+
+  describe("AllPaths", () => {
+    type res1 = Call<
+      // ^?
+      Objects.AllPaths,
+      {
+        shallow: string;
+        object: {
+          nested: boolean;
+        };
+        constant: true;
+        tuple: [0, 1];
+        union:
+          | { flag: true; ordinal: number }
+          | { flag: false; cardinal: string };
+        array: { inner: number }[];
+        conditional?: number;
+      }
+    >;
+    type test1 = Expect<
+      Equal<
+        res1,
+        | "object.nested"
+        | "object"
+        | "shallow"
+        | "constant"
+        | "tuple"
+        | "tuple.0"
+        | "tuple.1"
```

Comment:
shouldn't those be `tuple[0]` and `tuple[1]`?

Reply:
both should work, but makes sense to use brackets instead